Makefile.toml

```diff
@@ -63,7 +63,7 @@ dependencies = ["member_format", "member_clippy", "docker:build"]
 [tasks.access]
 workspace = false
 command = "docker-compose"
-args = ["-f", "./docker-compose.dev.yml", "exec", "--user", "steam", "valheim", "bash"]
+args = ["-f", ".`/docker-compose.dev.yml", "exec", "--user", "steam", "valheim", "bash"]
```

Comment:
(away from computer so cant test manually) Is this change intentional?

Reply:
lol rogue back tic :P ima fix that